src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -650,6 +651,20 @@ protected Owner createOwner(String key, String name) {
         return owner;
     }
 
+    protected Owner createOwnerEnt(String key) {
+        return this.createOwnerEnt(key, key);
+    }
+
+    protected Owner createOwnerEnt(String key, String name) {
+        Owner owner = TestUtil.createOwner(key, name);
+        owner.setId(null);
```

Comment:
you can use method chaining here to save some repetition:

```
Owner owner = TestUtil.createOwner(key, name)
  .setId(null)
  .setContentAccessMode(...);
```

Reply:
done